lib/models/translation.dart

```diff
@@ -20,14 +20,15 @@ class Translation {
   String? operator [](String? id) => assets[id];
   void operator []=(String id, String asset) => assets[id] = asset;
 
-  String node(Node? n) => this[n?.id] ?? '[❌NODE]';
+  String node(Node? n, [bool allowEmpty = false]) =>
```

Comment:
A boolean as a positional argument is almost never clear in its purpose. `node(n, true)` doesn't really say what this `true` means. Better change it to optional named argument